sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_receiver_async.py

```diff
@@ -41,6 +41,9 @@ class ServiceBusReceiver(collections.abc.AsyncIterator, BaseHandler, ReceiverMix
     """The ServiceBusReceiver class defines a high level interface for
     receiving messages from the Azure Service Bus Queue or Topic Subscription.
 
+    The two primary channels for message receipt are `receive()` to make a single request for messages,
+    and `for message in receiver:` to continuously receive incoming messages in an ongoing fashion.
```

Comment:
```suggestion
    and `async for message in receiver:` to continuously receive incoming messages in an ongoing fashion.
```

Reply:
Good catch.  Needs adjustment in aio session receiver too; will tweak that with the non-suggestion PR fixes momentarily.